platforms/core-configuration/base-diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/TaskReportTask.java

```diff
@@ -245,44 +252,63 @@ private List<RuleDetails> ruleDetailsFor(Project project) {
 
     private DefaultGroupTaskReportModel taskReportModelFor(Project project, boolean detail) {
         final AggregateMultiProjectTaskReportModel aggregateModel = new AggregateMultiProjectTaskReportModel(!detail, detail, getDisplayGroup(), getDisplayGroups());
-        final TaskDetailsFactory taskDetailsFactory = new TaskDetailsFactory(project);
 
-        final SingleProjectTaskReportModel projectTaskModel = buildTaskReportModelFor(taskDetailsFactory, project);
-        aggregateModel.add(projectTaskModel);
+        ProjectIdentity relativeProjectIdentity = ((ProjectInternal) project).getProjectIdentity();
```

Comment:
❔ What does `relative` mean in this context? The identity is always absolute. The `project` here can be any project (coming from `setProjects()`), not relative to the project owning this reporting task.

Reply:
I see, thanks for clarifying